lomap/tests/test_fsa.py

```diff
@@ -0,0 +1,50 @@
+from lomap.classes import Fsa
+
+
+def test_fsa():
+	specs1 = ['F a && F !b']
+	specs2 = ['F !a || F b || F !c']
+	specs3 = ['F a && F b && F c || F !a']
+	all_specs = [specs1, specs2, specs3]
+
+	for spec in specs1:
```

Comment:
For loop is not needed if you have only one element in the list. Put all specs in one list and loop over them.